packages/sanity/src/core/releases/components/BundleMenuButton/BundleMenuButton.tsx

```diff
@@ -118,7 +134,7 @@ export const BundleMenuButton = ({bundle, documentCount}: Props) => {
         >
           {bundleHasDocuments && (
             <Text data-testid="confirm-delete-body" muted size={1}>
-              This will also delete {documentCount} document version{documentCount > 1 ? 's' : ''}.
+              {t('delete.warning', {documentCount, count: documentCount})}
```

Comment:
nit pick, is it possible to use count as the variable in the translation?

```suggestion
              {t('delete.warning', {count: documentCount})}
```
```
  /** Warning for deleting a release that it will delete one document version */
  'delete.warning_one': 'This will also delete one document version.',
  /** Warning for deleting a release that it will delete multiple document version */
  'delete.warning_other': 'This will also delete {{count}} document versions.',
```

Reply:
Oh yeah definitely, that's how I've done it in other parts, clearly missed this one. I'll make this change 👍